browser/src/simulations/noop_scenario.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {handlePreferenceCheckbox, performLogin} from './login.js';
+import {postInChannel} from './post_in_channel.js';
+import {scrollInChannel} from './scrolling_in_channel.js';
+import type {BrowserInstance} from '../lib/browser_manager.js';
+
+// NoopScenario is a very basic scenario.
+// It just performs very basic actions in a loop. Such as:
+// Login, Post in channel, Scroll in channel and then repeat the process.
+export async function noopScenario({page, userId, password}: BrowserInstance, serverURL: string, runInLoop = true) {
+  if (!page) {
+    throw new Error('Page is not initialized');
+  }
+
+  try {
+    await page.goto(serverURL);
+    await handlePreferenceCheckbox(page);
+    await performLogin({page, userId, password});
+
+    // Runs the simulation atleast once and then runs it in continuous loop if simulationMode is true
+    do {
+      const scrollCount = runInLoop ? 40 : 3;
+      await postInChannel({page});
+      await scrollInChannel(page, 'sidebarItem_off-topic', scrollCount, 400, 500);
+      await scrollInChannel(page, 'sidebarItem_town-square', scrollCount, 400, 500);
+    } while (runInLoop);
```

Comment:
Typo, and should `simulationMode` be `runInLoop`?
```suggestion
    // Runs the simulation at least once and then runs it in continuous loop if runInLoop is true
    do {
      const scrollCount = runInLoop ? 40 : 3;
      await postInChannel({page});
      await scrollInChannel(page, 'sidebarItem_off-topic', scrollCount, 400, 500);
      await scrollInChannel(page, 'sidebarItem_town-square', scrollCount, 400, 500);
    } while (runInLoop);
```

Reply:
Fixed the typo